edx_sga/tasks.py

```diff
@@ -96,24 +141,38 @@ def _get_submissions_base_name(username, block_id, course_id):
     )
 
 
-def _get_dir_path(submissions_dir_name, locator=None):
+def _get_abs_local_path(submissions_dir_name, locator):
+    """
+    returns absolute directory path on local drive.
+
+    Args:
+        submissions_dir_name (str): name of folder
+        locator (BlockUsageLocator): BlockUsageLocator for the sga module
+    """
+    path = "{loc.org}/{loc.course}/{loc.run}/{submissions_dir_name}".format(
+        loc=locator,
+        submissions_dir_name=submissions_dir_name
+    )
+    return os.path.join(DATA_DIR, path)
+
+
+def _get_relative_dir_path(submissions_dir_name, locator):
     """
     returns directory path where we are saving files.
 
     Args:
         submissions_dir_name (str): name of folder
         locator (BlockUsageLocator): BlockUsageLocator for the sga module
     """
-    if locator:
-        return os.path.join(
-            DATA_DIR,
-            "{loc.org}/{loc.course}/{loc.run}/{submissions_dir_name}".format(
-                loc=locator,
-                submissions_dir_name=submissions_dir_name
-            )
-        )
+    path = "{loc.org}/{loc.course}/{loc.run}/{submissions_dir_name}".format(
+        loc=locator,
+        submissions_dir_name=submissions_dir_name
+    )
 
-    return os.path.join(DATA_DIR, submissions_dir_name)
+    if is_s3_default_storage():
+        return path
+
+    return os.path.join(DATA_DIR, path)
```

Comment:
This function is called `_get_relative_dir_path`, but when the storage type is not S3, this is an absolute path. This indicates the path to the submissions directory, right? How about we call it `_get_submissions_dir_path`?